benchmarks/profiler/utils/plot.py

```
@@ -252,3 +251,47 @@ def plot_decode_3d_surface(
+    d_itl, d_thpt = compute_parato(_d_itl, _d_thpt)
+
+    # convert to cost per thousand requests
+    p_ttft = np.array(p_ttft)
+    p_thpt = np.array(p_thpt)
+    d_itl = np.array(d_itl)
+    d_thpt = np.array(d_thpt)
+
+    tokens_per_user = []
+    cost = []
+    ttft = []
+    for _p_ttft, _p_thpt in zip(p_ttft, p_thpt):
```

Ruff (0.14.3)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)